src/test/java/uk/gov/hmcts/reform/idam/service/StaleUsersServiceTest.java

```diff
@@ -0,0 +1,158 @@
+package uk.gov.hmcts.reform.idam.service;
+
+import jakarta.ws.rs.core.Response;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.idam.parameter.ParameterResolver;
+import uk.gov.hmcts.reform.idam.service.remote.RestClient;
+import uk.gov.hmcts.reform.idam.service.remote.StaleUsersRequestParams;
+import uk.gov.hmcts.reform.idam.service.remote.responses.RoleAssignment;
+import uk.gov.hmcts.reform.idam.service.remote.responses.RoleAssignmentResponse;
+import uk.gov.hmcts.reform.idam.service.remote.responses.StaleUsersResponse;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyMap;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.nullable;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.springframework.http.HttpStatus.BAD_REQUEST;
+import static org.springframework.http.HttpStatus.INTERNAL_SERVER_ERROR;
+import static org.springframework.http.HttpStatus.OK;
+
+@ExtendWith(MockitoExtension.class)
+class StaleUsersServiceTest {
+    @Mock
+    RestClient client;
+
+    @Mock
+    private StaleUsersRequestParams requestParams;
```

Comment:
Do we use this?

Reply:
Good spot. No it is not used, though I was thinking about and I guess it stayed.